eve/methods/get.py

```diff
@@ -122,11 +122,18 @@ def _perform_aggregation(resource, pipeline, options):
     # implementation
 
     def parse_aggregation_stage(d, key, value):
-        for st_key, st_value in d.items():
-            if isinstance(st_value, dict):
-                parse_aggregation_stage(st_value, key, value)
-            if key == st_value:
-                d[st_key] = value
+        if isinstance(d, dict):
+            for st_key, st_value in d.items():
+                if isinstance(st_value, dict) or isinstance(st_value, list):
+                    parse_aggregation_stage(st_value, key, value)
+                if key == st_value:
+                    d[st_key] = value
+        if isinstance(d, list):
+            for st_idx, st_value in enumerate(d):
+                if isinstance(st_value, dict) or isinstance(st_value, list):
+                    parse_aggregation_stage(st_value, key, value)
+                if key == st_value:
+                    d[st_idx] = value
```

Comment:
Let us see what @nicolaiarocci says, but maybe it is worth to report it in the documentation related to the aggregation endpoint